service_capacity_modeling/tools/generate_missing.py

```diff
@@ -29,7 +29,9 @@ def build_command(family: str, params: Dict[str, Any], output_path: Path) -> lis
         cmd.extend(["--io-latency-curve", params["io_latency_curve"]])
 
     if params.get("cpu_ipc_scale") is not None:
-        cmd.extend(["--cpu-ipc-scale", str(params["cpu_ipc_scale"])])
+        cpu_ipc_scale = float(params["cpu_ipc_scale"])
+        rounded_cpu_ipc_scale = float(f"{cpu_ipc_scale:.2f}")
```

Comment:
I used two decimal places here vs two sig figs, since we normally get two sig figs as an uplift, (eg: 9%, 12%, 15%), and this data is stored as a multiplier, where this would be 1.09, 1.12, 1.15.